Makefile

```diff
@@ -111,7 +111,7 @@ clean:
 	-rm -f calico_containers/calico-node.tar
 	-rm -f calico_containers/routereflector.tar
 	-docker rm -f calico-build
-	-docker rm -f calico-node
+	-docker rm -f calico_node
```

Comment:
The Docker image hasn't changed name has it?  Shouldn't this still be "calico-node"?